src/app/components/views/thumbnail/preview.component.html

```diff
@@ -17,13 +17,25 @@
       (click)="videoClick.emit($event)"
       [ngStyle]="{ height: imgHeight + 'px' }"
     >
+
+      <!-- Show one thumbnail if there is only one -->
       <div
-        *ngIf="folderThumbPaths[0]"
-        class="folder-item"
+        *ngIf="!folderThumbPaths[1] && folderThumbPaths[0]"
```

Comment:
FYI, since you're treating this as an `if / else`, an approach you can do with _Angular_ is similar to line 12 above
https://github.com/whyboris/Video-Hub-App/pull/350/files#diff-2dc1e1902eca723eb5d284832a0502d4R14
`<el *ngIf="condition; else selector">` and then `<el #selector>` 

I'll likely update the code into this pattern after merging 👌 

Thank you for the excellent contribution 🤝

Reply:
Just a minor change: https://github.com/whyboris/Video-Hub-App/pull/356/commits/2be2f1ba659ded26b4bea4ec0667418a4a903e9a 

Cheers 🙂